mainnet_v1/bridged-token-list.json

```diff
@@ -8,7 +8,9 @@
     "erc20Info": {
       "sudtScriptArgs": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "ethAddress": "0x7538C85caE4E4673253fFd2568c1F1b48A71558a"
-    }
+    },
+    "UAN": "pCKB.gw|gb.ckb",
```

Comment:
Should we name it `CKB.gw|gb.ckb` since the bridged token is `CKB` but not `pCKB`

Reply:
The decimals of CKB is 8, while the decimals of pCKB is 18.